src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
```

Comment:
This `Service` keyword indicates that there is business logic here
```suggestion
public class FileReaderServiceImpl implements FileReaderService {
```

Alternatively, you can rewrite as in the following logic
```
    @Override
    public List<String> readFromFile(String fromFilePath) {
        try {
            return Files.readAllLines(Path.of(fromFilePath));
        } catch (IOException e) {
            throw new RuntimeException("Can't read data from file "
            + fromFilePath + " !", e);
        }
    }
```